Gemfile.lock

```diff
@@ -43,9 +45,18 @@ GEM
     builder (3.2.2)
     cf-uaa-lib (3.2.5)
       multi_json
+    cf-uaac (3.1.6)
```

Comment:
Could we use the most recent gem? The latest one is 3.2.0: https://rubygems.org/gems/cf-uaac